skopt/tests/test_parameter.py

```diff
@@ -0,0 +1,86 @@
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_equal
+
+from skopt.parameter import sample_points
+from skopt.parameter import Uniform
+from skopt.parameter import Integer
+from skopt.parameter import Categorical
+
+
+def check_distribution(Dist, vals, random_val):
+    x = Dist(*vals)
+    assert_equal(x.rvs(random_state=1), random_val)
+
+
+def test_distributions():
+    yield (check_distribution, Uniform, (1., 4.), 2.668088018810296)
+    yield (check_distribution, Uniform, (1, 4), 2.668088018810296)
+    yield (check_distribution, Integer, (1, 4), 2)
+    yield (check_distribution, Integer, (1., 4.), 2)
+    yield (check_distribution, Categorical, ('a', 'b', 'c', 'd'), 'b')
+    yield (check_distribution, Categorical, (1., 2., 3., 4.), 2.)
```

Comment:
Might be best to check that Uniform.rvs returns a float, that Integer.rvs returns and that Categorical(["a", "b", ...]) returns a string, all within the given bounds/set. What do you think?